config/app.default.php

```diff
@@ -9,7 +9,7 @@
      * Development Mode:
      * true: Errors and warnings shown.
      */
-    'debug' => true,
+    'debug' => env('DEBUG', true),
```

Comment:
For instance having such a thing without a prefix is no good.
For the databases I can see the reason to not prefix things *in some edge cases*, but I would by convention I'd still prefix those, too.